firebase-installations-interop/src/main/java/com/google/firebase/installations/FirebaseInstallationsApi.java

```diff
@@ -24,14 +28,32 @@
  */
 public interface FirebaseInstallationsApi {
 
+  /**
+   * Specifies the options to get a FIS AuthToken. Default value of AuthTokenOption =
+   * DO_NOT_FORCE_REFRESH.
+   */
+  @IntDef({DO_NOT_FORCE_REFRESH, FORCE_REFRESH})
+  @Retention(SOURCE)
+  @interface AuthTokenOption {}
+  /**
+   * AuthToken is not refreshed until requested by the developer or if one doesn't exist, is expired
+   * or about to expire.
+   */
+  int DO_NOT_FORCE_REFRESH = 0;
+  /**
+   * AuthToken is forcefully refreshed on calling the {@link
+   * FirebaseInstallationsApi#getAuthToken(int)}.
+   */
+  int FORCE_REFRESH = 1;
+
   /**
    * Async function that returns a globally unique identifier of this Firebase app installation.
    * This is a url-safe base64 string of a 128-bit integer.
    */
   Task<String> getId();
 
   /** Async function that returns a auth token(public key) of this Firebase app installation. */
-  Task<InstallationTokenResult> getAuthToken(boolean forceRefresh);
```

Comment:
Sorry I was not following what you may have changed.  Why do we change to return string instead of InstallationTokenResult?
We decided to return a struct in Firebase API review go/fis-sdk-review

Reply:
https://docs.google.com/document/d/1ib_AMf959UAPVyrzcnYMIFsxV2dMCzl5ZPG-8qHTH1M/edit#heading=h.ankqhj32c8xa

Reply:
https://github.com/firebase/firebase-js-sdk/blob/master/packages/installations/src/functions/get-token.ts

ios and js sdk returns String too

Reply:
Well, 
1) the Android SDK design doc was written before the SDK review, I should change it but I didn't.
2) the iOS SDK and web SDK work started before the Android SDK review, we decided to do the struct in Android, and later change the iOS / web SDK to use the struct as well.

Reply:
Oh, got it. Makes sense. Rayo mentioned to be consistent with js-sdk as its already out.

I ll update it to a struct.

Reply:
Updated.